app/app.go

```diff
@@ -359,7 +359,7 @@ func NewStrideApp(
 		appCodec, keys[slashingtypes.StoreKey], &stakingKeeper, app.GetSubspace(slashingtypes.ModuleName),
 	)
 	app.CrisisKeeper = crisiskeeper.NewKeeper(
-		app.GetSubspace(crisistypes.ModuleName), invCheckPeriod, app.BankKeeper, authtypes.FeeCollectorName,
+		app.GetSubspace(crisistypes.ModuleName), 100, app.BankKeeper, authtypes.FeeCollectorName,
```

Comment:
Would prefer to put this in a setup/config/sh file. Does anyone know how to handle that?

Reply:
can we add that as a network param?

Reply:
If that'd make it up to the validators, I may just leave it here for now?

Reply:
yeah sorry, added this comment before I noticed it was a startup flag